modules/network-observability-operator-install.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-operator-installation_{context}"]
+
+= Installing the Network Observability operator
+You can install the Network Observability operator using the {product-title} web console Operator Hub.
+
+.Procedure
+ 
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+//In the Operator Hub on the test console Julian provided perms for, its NetObserv Operator, with a capital O. 
+.. Choose  *Network Observability operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
+.. Navigate to *Operators* -> *Installed Operators*. Under Provided APIs for Network Observability, select the *Flow Collector* resource.
+. Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
++
+- *Sampling* : Specify a sampling size that will not consume too many resources. For more information, see the FlowCollector custom resource configuration parameters reference table. 
```

Comment:
```suggestion
- *Sampling* : Specify a sampling size for flows, lower sampling sizes will have higher impact on resource utilization. For more information, see the FlowCollector custom resource configuration parameters reference table. 
```

Reply:
+1; maybe also mention where to find this field (it's in the Agent > ebpf section)